app/api/config/react.py

```diff
@@ -1,7 +1,2 @@
-from shared.mapping.models import Status
-
-
 def react(request):
-    return {
-        "status": [{"id": id, "label": label} for id, label in Status.choices],
-    }
+    return {}
```

Comment:
This is a breaking change, but I guess the PR is.. as in this is the first piece of functionality that will actually break the legacy React app. 
We should explore how much effort it would take to enable it to continue to work and avoid a breaking change.

Reply:
I see. Reverted with some relevant updated.